test_ngi_config.yaml

```diff
@@ -0,0 +1,97 @@
+# This file should be placed under $HOME/.ngiconfig/ngi_config.yaml
+# or its path exported as the environment variable NGI_CONFIG
+
+database:
+    # SQLite file to know where things are happening (state machine to back up Charon for network failure)
+    # -usually it is at /proj/a2014205/ngi_resources/record_tracking_database_nestor.sql
+    # -if you are trying to work without one, it is going to create an empty schema
+    record_tracking_db_path: /home/szilva/glob/private/sb/tracking/record_tracking_database.sql
```

Comment:
nice to have a test but then better if you can make this as template. @sylvinite we discuss about this in the morning, maybe your input is important